application/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 
 # Helm chart Version
 
-version: 4.2.12
+version: 4.2.13
```

Comment:
Can you please revert this change? The chart version will be automatically bumped.

Reply:
done ;)